requirements.txt

```diff
@@ -20,3 +20,4 @@ sphinx-jsonschema~=1.16
 pytest-console-scripts~=1.1
 pre-commit==2.10.1
 scipy
+python-opencv
```

Comment:
if required by Resample and we get rid of Resample, this might not be needed anymore

Reply:
It can be changed with a `scikit-image` library, which already exists in the `ivadomed` requirements.

Reply:
I removed the OpenCV dependency.